src/components/ReportingConfig/SFTPDeliveryMethodForm.jsx

```diff
@@ -9,11 +9,8 @@ const SFTPDeliveryMethodForm = ({ invalidFields, config, handleBlur }) => {
     <>
       <div className="row">
         <div className="col">
-          <ValidationFormGroup
-            for="sftpHostname"
-            helpText="The host to deliver the report too"
-            invalidMessage="Required. Hostname cannot be blank"
-            invalid={invalidFields.sftpHostname}
+          <Form.Group
+            isInvalid={invalidFields.sftpHostname}
           >
             <label htmlFor="sftpHostname">SFTP Hostname</label>
             <Input
```

Comment:
As depicted in [ENT-8329](https://2u-internal.atlassian.net/browse/ENT-8329), `Input` is deprecated. In this case, should be migrated to `Form.Control`.

**Paragon Docs**

* [`Input`](https://paragon-openedx.netlify.app/components/input/)
* [`Form.Control`](https://paragon-openedx.netlify.app/components/form/form-control/)